portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,14 +26,33 @@
 public class DataServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String[] wordArray = {"Hey!", "Check", "here!"};
+    /**
```

Comment:
Given that we use version control system (e.g., Git in this case), the convention is that we either keep the code we want, or remove it directly because (1) if you want it later, it's already versioned (see "git log" sub-command to retrieve the logs of commit history); and (2) readers don't need to read the code that is not in effect.

Reply:
oh! I didn't know that, thanks

Reply:
No problem.

I could share my own rule of thumb. Sometimes if I can't decide if a piece of code should look the way it is, I would pretend I am a newcomer to a codebase, and I don't have very good idea what's going on, would the code be clear to me. We stress readability very much because code is read more than written.